### ISSUE TITLE
A third argument for number delimiter

### ISSUE THREAD

**Issue by denydias** [chrisullyott/php-filesize#2]
This is a tiny beautiful package! So useful! Thanks a lot for it, @chrisullyott!

It would be great to add a third optional argument to `FileSize()` so one could choose the desired number delimiters (eg. 2.144.57 KB versus 2.144,57 KB). Something like:

```php
$size = new FileSize(10921134, 10); // Readme example
echo $size->asAuto(); // '10.92 MB'

$size = new FileSize(10921134, 10, true); // This is the FR on this issue
echo $size->asAuto(); // '10,92 MB'
```

As [there's currently no standards ruling decimal separators](https://en.wikipedia.org/wiki/Decimal_separator#Current_standards) (!), a simple optional `bool` argument would suffice and should not break anything for anyone.

Thanks in advance!

**Comment by chrisullyott** [comment]
@denydias Nice suggestion! 

I think there might be room for a number formatting functionality, something like what [NumberFormatter](https://www.php.net/manual/en/class.numberformatter.php) helps with. The user could be working with existing sizes in one format, but then desiring the output to be in a different format.

That said, there should probably be format knowledge both on the parsing side and on the output side.

**Comment by chrisullyott** [comment]
@denydias This has been added in [this release](https://github.com/chrisullyott/php-filesize/releases/tag/v4.2.0).

**Comment by denydias** [comment]
Thanks a bunch, @chrisullyott!

**Comment by denydias** [comment]
Just implemented 4.2. Worked like a charm!

![Screenshot_20211017_034551](https://user-images.githubusercontent.com/948202/137615072-13a84aae-cc7a-4ab2-9711-5fc2f5768129.png)
